docs/installation/development.md

```diff
@@ -20,19 +20,75 @@ Tilt is a great way to run OpenCost on a local or remote Kubernetes environment.
+9. As you update code, `tilt_config.json`, or `tilt-values.yaml`, Tilt will automatically watch relevant source code and recompile, build, deploy, and run tests.
+10. `tilt down` will tear down resources created, otherwise they will continue to run indefinitely.
+11. `kind delete` cluster will delete the local cluster, if desired.
+
+## Tilt Command Options
+
+### via `tilt_config.json`
+The best way to configure OpenCost to use tilt is with a `tilt_config.json` file in the root of the OpenCost repository. The file must live there; a symlink is thebest alternative if you wish to store it elsewhere. This file is automatically reloaded in tilt when updated. Omitted values will use defaults.
+Example file:
+```JSON
+{
+  "arch":"amd64",
+  "cloud-integration":"../cloud-integration.json"
+}
+```
+
+### via CLI
+These options can also be placed after `tilt up --`.
+Example command:
+`tilt up -- --arch arm64 --docker-repo dockerhub-username --cloud-integration ../cloud-integration.json --service-key ../service-key.json`
+Options defined from the CLI will override those from `tilt_config.json`.
+
+### Options
+- `arch`
+  Value: `amd64`(default),`arm64`
+- `cloud-integration`
+  Value: Path to a JSON file for cloud integrations
+  Optional. Defaults to an empty string.
+  This is the path to the JSON file to use for the `cloud-integration` secret. [See configuration documentation here.](../configuration)
+- `delve-continue`
+  Value: `true`(default) or `false`
+  Optional, defaults to true. Determines if the `--continue` flag is used with delve. By default, this will let applications start without a debugger attached. `false` will wait for a debugger to attach before starting.
+- `docker-repo` 
+  Value: Docker Hub username or repository, defaults to an empty string.
+  Required for a remote cluster, optional for local. 
+  Optional. This is the CPU architecture of the desired node(s) on the cluster.
+  The value is prepended to the docker image names (the -t flag in a Docker build command). [See Docker Documentation](https://docs.docker.com/engine/reference/commandline/build/#tag).
+- `helm-values`
+  Optional. Defaults to `./tilt-values.yaml`
+  This is the path to the values.yaml file that will be used for the OpenCost helm chart.
+- `service-key`
+  Value: Path to a JSON file for a service key
+  Optional. Defaults to an empty string.
+  This is the path to the JSON file to use for the `service-key` secret. [See configuration documentation.](../configuration)
+- `port-costmodel`
+  Optional. Defaults to 9003.
+  This is the port forwarded to localhost for the costmodel (back end).
+  Note: In `tilt_config.json`, this is a string and must be in quotes, since Tilt options must be a bool or a string.
+- `port-debug`
+  Optional. Defaults to 40000.
+  This is the port forwarded to localhost for delve.
+  Note: In `tilt_config.json`, this is a string and must be in quotes, since Tilt options must be a bool or a string.
+- `port-prometheus`
```

Comment:
I think Prometheus is listening on 80 inside the OpenCost cluster, does this forward 80 to 9080?

Reply:
Correct. The only configurable port through tilt is the 'local' side.